config/community-relays.json

```diff
@@ -6,5 +6,11 @@
             "host": "relays.shareslake.network",
             "port": 3001
         },
+        {
+            "name": "Spire",
+            "valency": 2,
```

Comment:
It seems the valency should be `1` in your pool. When I try to set valency to `2` it fails to start the subscriptions, but with `1` it works.